queries.sql

```diff
@@ -22,4 +22,40 @@ SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE name not like 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 and 17.3; 
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 and 17.3; 
+
+/*  Transactions  */
+
+/* Stage 1 */
+BEGIN;
+UPDATE animals SET species='unspecified';
+ROLLBACK;
+
+/* Stage 2 */
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE name NOT LIKE '%mon';
+COMMIT;
+
+/* Stage 3 DELETE */ 
+BEGIN;
+DELETE FROM animals;
```

Comment:
- [ ] I think this query in not completed yet so kindly complete it `delete all records in the animals table`

so it could be something like this:
> DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;